src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1750,19 +1750,27 @@ private IVariableDeclarationGroupOperation CreateBoundMultipleLocalDeclarationsB
             bool declarationIsImplicit = boundMultipleLocalDeclarations.WasCompilerGenerated;
             IVariableDeclarationOperation multiVariableDeclaration = new CSharpLazyVariableDeclarationOperation(this, boundMultipleLocalDeclarations, _semanticModel, declarationSyntax, null, default, declarationIsImplicit);
 
-            // If this is a using declaration, work out the declaration kind
-            VariableDeclarationKind declKind = boundMultipleLocalDeclarations is BoundUsingLocalDeclarations usingDecl
-                                               ? usingDecl.AwaitOpt is object
-                                                    ? VariableDeclarationKind.AsynchronousUsing
-                                                    : VariableDeclarationKind.Using
-                                               : VariableDeclarationKind.Default;
-
             ITypeSymbol type = null;
             Optional<object> constantValue = default(Optional<object>);
             // If the syntax was the same, we're in a fixed statement or using statement. We make the Group operation implicit in this scenario, as the
-            // syntax itself is a VariableDeclaration
-            bool isImplicit = declarationGroupSyntax == declarationSyntax || boundMultipleLocalDeclarations.WasCompilerGenerated;
-            return new VariableDeclarationGroupOperation(ImmutableArray.Create(multiVariableDeclaration), declKind, _semanticModel, declarationGroupSyntax, type, constantValue, isImplicit);
+            // syntax itself is a VariableDeclaration. We do this for using declarations as well, but since that doesn't have a separate parent bound
+            // node, we need to check the current node for that explicitly.
+            bool isImplicit = declarationGroupSyntax == declarationSyntax || boundMultipleLocalDeclarations.WasCompilerGenerated || boundMultipleLocalDeclarations is BoundUsingLocalDeclarations;
```

Comment:
Nit: you could do the cast from line 1761 here and cache the result, rather than looking at it twice

Reply:
I tried this, but then because `var isUsingDeclaration = boundMultipleLocalDeclarations is BoundUsingLocalDeclarations usingDecl;` doesn't definitely assign `usingDecl` I had to break it out further into an `as` and null check, and at that point I just left it like this.